xml/System.Diagnostics.Metrics/ObservableInstrument`1.xml

```diff
@@ -80,8 +80,8 @@ This class supports only the following generic parameter types: <xref:System.Byt
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>A property tells if the instrument is an observable instrument. This property will return <see langword="true" /> for all metrics observable instruments.</summary>
-        <value>To be added.</value>
+        <summary>A property tells if the instrument is an observable instrument.</summary>
+        <value><see langword="true" /> for all metrics observable instruments; <see langword="false"/> for all other types of instruments.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the instrument is metrics-observable; <see langword="false"/> otherwise.</value>
```